CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Grafana Mimir
 
 * [ENHANCEMENT] Ingester: Add support for exporting native histogram cost attribution metrics (`cortex_ingester_attributed_active_native_histogram_series` and `cortex_ingester_attributed_active_native_histogram_buckets`) with labels specified by customers to a custom Prometheus registry. #10892
+* [BUGFIX] OTLP: Fix response body and Content-Type header to align with spec. #10852
```

Comment:
Would it make sense to backport this bugfix into 2.16 (separately), since we're that earlier in the release?

Reply:
I'm inclined to leave it out given the number of files touched in the PR and that we haven't ported it to r334 and so aren't testing in a meaningful environment this week. The originating bug was opened by Fiona so it doesn't seem super urgent for OSS users.